test/smoke.spec.js

```diff
@@ -154,4 +160,49 @@ describe("Smoke tests", () => {
       });
     });
   });
+
+  describe("daemon", () => {
+    beforeEach(() => mkdirp(testOutputDir));
+    afterEach(() => rimraf.sync(testOutputDir));
```

Comment:
Also, add any test outputs to `.gitignore`. There may be a test failure in development and we don't want a footgun that could accidentally lead to that getting committed.